### PR TITLE
Add Buoyant to ADOPTERS.md

### PR DESCRIPTION
**Pull request by wmorgan**
Blatant self-promotion / selflessly doing my part to demonstrate adoption.

### CODE REVIEW

**Comment by mosesn** [comment]
Can you add a trailing slash for consistency?  Also, y u no TLS???

**Comment by wmorgan** [comment]
Added!

(TLS: I know right! Still waiting on our ops org...)

**Comment by travisbrown** [comment]
Pulling this in to be merged.

File: ADOPTERS.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ URLs if possible) and [submit a pull request][1]!
 
 * [Attendify](https://attendify.com)
 * [Box](https://www.box.com/)
+* [Buoyant](http://buoyant.io/)
```
Comment (dschobel) [inline ADOPTERS.md]: pretty sure it's spelled Buoyánt

Comment (caniszczyk) [inline ADOPTERS.md]: yes, can someone fix this?

Comment (wmorgan) [inline ADOPTERS.md]: Buøyant is actually our preference (in honor of Finagle's Norse heritage). But we're fine with whatever.

**Comment by travisbrown** [comment]
[Merged](https://github.com/twitter/finagle/commit/0475bd888d7c5c7007a28f90e4ddcef990617249).

**Pull request closed without merging**